clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/recordreader/vson/VeniceVsonFileIterator.java

```diff
@@ -27,7 +27,7 @@ public VeniceVsonFileIterator(FileSystem fs, Path hdfsPath, VeniceVsonRecordRead
       try {
         fileReader = new SequenceFile.Reader(fs, hdfsPath, new Configuration());
       } catch (IOException e) {
-        LOGGER.info("Path: {} is not a sequence file.", hdfsPath.getName());
```

Comment:
Why not have both log and throw?

Reply:
That's because the expectation when throwing an exception is either the component catching it will decide how it would like to handle thigs, or it will get logged with a stacktrace